### PR TITLE
Team Anchor -  Roles & Permissions + Invites to Org

### PR DESCRIPTION
**Pull request by Froches**
# Description

**Closes #issue_number_here**

# Changes proposed

## What were you told to do?
Roles & Permissions and Invite to Org

## What did you do?
I created the members page according to the figma design given for the members page for the "Invite to Org" feature assigned to me.

# Check List (Check all the applicable boxes)

🚨Please review the [contribution guideline](CONTRIBUTING.md) for this repository.

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR is clear and explains the approach.
- [x] I am making a pull request against the **dev branch** (left side).
- [x] My commit messages styles matches our requested structure.
- [x] My code additions will fail neither code linting checks nor unit test.
- [x] I am only making changes to files I was requested to.

# Screenshots/Videos
Screenshot 2024-07-31 at 3 36 13 AM
Screenshot 2024-07-31 at 3 38 01 AM

### CODE REVIEW

**Review by ikennarichard — DISMISSED** [review]
Completed

**Review by raji-ayyub — DISMISSED** [review]
checked

File: src/app/dashboard/(user-dashboard)/settings/organization/members/page.tsx [hunk 1]
```diff
@@ -1,5 +1,211 @@
...
+    id: 5,
+    name: "Chad Bosewick",
+    email: "ChadBoseW@gmail.com",
+    role: "Admin",
+  },
+];
+
+const activeMembers: number = memberData.length;
+
+const Menbers = () => {
```
Comment (billmal071) [inline src/app/dashboard/(user-dashboard)/settings/organization/members/page.tsx]: Rename to members.

File: src/app/dashboard/(user-dashboard)/settings/organization/members/page.tsx [hunk 2]
```diff
@@ -1,5 +1,211 @@
...
+          <CustomButton variant="primary">Export CSV</CustomButton>
+        </div>
+      </div>
+      <InviteMemberModal show={isModalOpen} onClose={handleModalClose} />
+    </div>
+  );
 };
 
-export default page;
+export default Menbers;
```
Comment (billmal071) [inline src/app/dashboard/(user-dashboard)/settings/organization/members/page.tsx]: Same here. Members.

**Review by billmal071 — CHANGES_REQUESTED** [review]

**Review by billmal071 — APPROVED** [review]
LGTM

**Review by ikennarichard — APPROVED** [review]

**Pull request merged**